rest-api-model-server/src/main/kotlin/org/modelix/sample/ModelClientConfiguration.kt

```diff
@@ -0,0 +1,26 @@
+package org.modelix.sample
+
+import io.smallrye.config.ConfigMapping
+import io.smallrye.config.WithDefault
+import java.net.URI
+import java.util.*
+
+/**m
+ * Describes configuration properties required to access a model-server instance using the client code.
+ *
+ * Cf. https://quarkus.io/guides/config-mappings for the underlying Quarkus configuration mechanism.
+ */
+@ConfigMapping(prefix = "modelix.client")
+interface ModelClientConfiguration {
+    @WithDefault(value = "http://0.0.0.0:28101")
```

Comment:
Can we configure this from the outside or `application.properties`?

Reply:
Sure, this is the default configuration mechanism and a whole bunch of value sources are available: https://quarkus.io/guides/config-reference#configuration-sources